overlay.d/05core/usr/lib/dracut/modules.d/40ignition-ostree/ignition-ostree-transposefs.sh

```diff
@@ -202,6 +244,23 @@ case "${1:-}" in
             mkdir "${saved_prep}"
         fi
         ;;
+    autosave-xfs)
+        should_autosave=$(should_autosave_rootfs)
+        if [ "${should_autosave}" = "1" ]; then
```

Comment:
```suggestion
        if [ "${should_autosave}" == "1" ]; then
```

It always throws me off that a single `=` works.

You don't need to change this, I'm just calling it out.